pyproject.toml

```diff
@@ -0,0 +1,116 @@
+# build-system
+# - ref: https://peps.python.org/pep-0517/
+#
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+
+# project
+# - ref 1: https://peps.python.org/pep-0621/
+# - ref 2: https://hatch.pypa.io/latest/config/metadata/#project-metadata
+#
+[project]
+name = "jupyterhub-pytest-plugin"
+description = "A reusable JupyterHub pytest plugin"
+readme = "README.md"
+requires-python = ">=3.5"
+license = {file = "LICENSE"}
+keywords = ["jupyterhub", "pytest", "plugin"]
+authors = [
+    {name = "Jupyter Contributors", email = "jupyter@googlegroups.com"},
+]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Programming Language :: Python :: 3",
+]
+# TODO: List the packages and pin them to a version that your project depends on
+dependencies = [
+    # NOTE: If lower bounds are updated, also update our test for the lower
+    #       bounds in .github/workflows/test.yaml.
+    "jupyterhub>=2.0.0",
+]
+dynamic = ["version"]
+
+# TODO: List the packages and pin them to a version that the tests in your project depend on
+[project.optional-dependencies]
+test = [
+    "pytest>=5.4",
+    "pytest-cov",
+    "pytest-asyncio>=0.17",
+]
```

Comment:
I believe it makes more sense to not actually pin to these minimum versions of these packages as we don't have such a restriction. Instead, make the latest version of them (you can check that out on https://pypi.org), the min version required.

I know this is coming from the template repository, so do you want to open a PR there also, to remove these versions?

Reply:
Yes, I will open a PR in the template repo concerning this.

Reply:
Done